CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Next Version
 
+#### Added
+- Add shared breakpoints support [#693](https://github.com/yonaskolb/XcodeGen/pull/693) @myihsan
```

Comment:
```suggestion
- Add shared breakpoints support [#693](https://github.com/yonaskolb/XcodeGen/pull/693) @myihsan @alexruperez
```

Reply:
Fixed.
Sorry for this terrible mistake.